DESCRIPTION

```diff
@@ -27,11 +27,18 @@ Imports:
     AzureRMR,
     AzureStor,
     cli,
+    data.table,
     DBI,
+    dplyr,
     duckdb,
     EpiNow2 (>= 1.4.0),
-    rlang
+    jsonlite,
+    rlang,
+    rstan,
+    tidybayes
+Additional_repositories:
+    https://mc-stan.org/r-packages/
 URL: https://cdcgov.github.io/cfa-epinow2-pipeline/
 Depends:
-    R (>= 2.10)
+    R (>= 3.50)
```

Comment:
This change is because of the new rds file in the tests. The rds file changed in R v3.5.0